pkg/sql/plan/query_builder.go

```diff
@@ -2063,6 +2064,22 @@ func (builder *QueryBuilder) buildTable(stmt tree.TableExpr, ctx *BindContext, p
 						return 0, err
 					}
 					builder.qry.Nodes[nodeID].FilterList = accountFilterExprs
+				} else if dbName == catalog.MO_SYSTEM_METRICS && tableName == catalog.MO_METRIC {
```

Comment:
This is almost for sure the wrong way to put in extra filter conditions.     There is nothing special about mo_metrics and mo_statement table.    We only have 3 special tables, cannot add more like this.